content/en/docs/pluggable-components/auditing-logs.md

```diff
@@ -45,7 +45,7 @@ By default, KubeKey will install Elasticsearch internally if Auditing is enabled
       elasticsearchDataVolumeSize: 20Gi    # The volume size of Elasticsearch data nodes.
       logMaxAge: 7                     # Log retention day in built-in Elasticsearch. It is 7 days by default.
       elkPrefix: logstash              # The string making up index names. The index name will be formatted as ks-<elk_prefix>-log.
-      externalElasticsearchUrl: # The URL of external Elasticsearch.
+      externalElasticsearchHost: # The URL of external Elasticsearch.
```

Comment:
externalElasticsearchHost: # The Host of external Elasticsearch.
is better

Reply:
Done.